src/site/content/en/blog/app-shortcuts/index.md

```diff
@@ -0,0 +1,233 @@
+a relative URL, the base URL will be the URL of the web app manifest.
+
+### icons (optional)
+
+An array of image resource objects. Each object must
+include the `src` and a `sizes` property. The `type` of image is optional.
+
+SVG files are not supported at the time of writing, use PNG instead.
+
+If you want pixel-perfect icons, provide them in
+increments of 48dp. Otherwise, it is recommended that you use a 192x192 pixels
+icon.
+
+## Test your app shortcuts
+
+To verify your app shortcuts are setup correctly, use the **Manifest** pane in the
+**Application** panel of DevTools.
+
+<figure class="w-figure">
+  <img src="./app-shortcuts-devtools.png" alt="Screenshot of app shortcuts in DevTools">
+  <figcaption class="w-figcaption">App shortcuts shown in DevTools</figcaption>
+</figure>
+
+This pane provides a human-readable version of many of your manifest's
+properties, including app shortcuts. It makes it easy to verify that all of the
+shortcut icons, if provided, are loading properly.
+
+{% Aside %}
+DevTools support for app shortcuts is available in Chrome 84 and Edge 84 and later.
+{% endAside %}
+
+App shortcuts may not be available right away to all users because PWA updates
+are capped to once a day.  You can find more details in this [article].
+
+## Best practises
+
+### Order app shortcuts by priority
+
+You are encouraged to order app shortcuts by priority, with the most critical
+app shortcuts appearing first in the `shortcuts` array as the limit on the
+number of app shortcuts displayed varies depending on the platform. Chrome and
+Edge on Windows for instance limit the number of app shortcuts to 10 while
+Chrome for Android only takes the first 4 app shortcuts into account.
+
+### Use distinct app shortcut names
+
+You should not rely on icons to differentiate app shortcuts as they may not
+always be visible. For instance, macOS doesn't support icons in the dock
+shortcuts menu. Use distinct names for each app shortcut.
+
+### Measure app shortcuts usage
+
+You should annotate app shortcuts `url` entries like you would do with
+`start_url` for analytics purposes (.e.g. `url:
+"/my-shortcut?utm_source=homescreen"`).
+
+## Browser support
+
+App shortcuts are available in Chrome 84 on Android. On Windows, you can
+experiment with them on both Chrome 84 and Edge 84 by enabling the flag
+`about://flags/#enable-desktop-pwas-app-icon-shortcuts-menu`. More desktop
+platform support will follow.
+
+## Trusted Web Activity support
+
+[Bubblewrap], the recommended tool to build Android apps that use [Trusted Web
+Activity], reads app shortcuts from the web app manifest and automatically
+generates the corresponding configuration for the Android app.
+
+For developers integrating Trusted Web Activity manually into their Android
+application, [Android app shortcuts] can be used to implement the same
+behaviors.
+
+## Sample
+
+Check out the app shortcuts [sample] and its [source].
```

Comment:
Why not embed the Glitch into this post? 

https://web.dev/handbook/web-dev-components/#glitches

Reply:
Because it is not actionable. Meaning user has to install the app, and there's not much to see to be honest.

Reply:
But they can immediately browse the source code of the Glitch?

Reply:
I've added a glitch embed in latest patch.